tests/training/models/amun/test_amun_has_decoder_c_tt.sh

```diff
@@ -15,7 +15,7 @@ $MRT_MARIAN/marian \
 
 test -e decoder_c_tt/model.npz
 
-python $MRT_MARIAN/../scripts/contrib/model_info.py -m decoder_c_tt/model.npz > decoder_c_tt.out
+$MRT_MARIAN/../scripts/contrib/model_info.py -m decoder_c_tt/model.npz > decoder_c_tt.out
```

Comment:
Does this script work for you as expected with python 3+? If I remember correctly, I couldn't make the output identical for both python 2.7 and 3.6 and that was the reason why python 2.7 was used here. Or maybe this is a difference in numpy versions between both versions of python on our servers.

Reply:
I have not tried it, sorry. I would not expect that the Python version would affect numeric results, esp. since Python is only used for loading and comparing files (or is it not?).

I made this change because tests failed on my machine, which no longer has a maintained python2; it only exists as needed by Ubuntu (18) and therefore has no numpy nor yaml.

The script in the line in question has a hash-bang referencing python3, but was explicitly invoked via "python" without version, invoking python2 in my machine. Since the script explicitly states the version, this was an obvious bug (if an inconsistency like this is intentional, the author should have added a comment), and fixing it would eliminate a dependency (on python2). So I opted to fix it, rather than work around it by installing the needed python2 packages.

Reply:
No worries and many thanks for pointing this out. It works now. Thanks!